HCCGo/app/js/dbService.js

```diff
@@ -40,6 +42,8 @@ dbService.service('dbService', ['filePathService', function(filePathService) {
     getSubmittedJobsDB: function() {
       return submittedJobsDB;
     }
+
+
```

Comment:
I was looking into making the preferences a dbService, didn't catch it in the commit